man/man4/zfs.4

```diff
@@ -73,6 +73,17 @@ When set to
 .Sy 0
 the array is dynamically sized based on total system memory.
 .
+.It Sy dbuf_evict_parallel Ns = Ns Sy 0 Pq uint
+When set to 1, ZFS will use up to
+.Sy dbuf_evict_threads
+threads to evict dbuf data in parallel, improving the responsiveness
+of ZFS to memory pressure.
+.
+.It Sy dbuf_evict_threads Ns = Ns Sy 0 Pq uint
+Sets the maximum number of dbuf eviction threads to be used.
+When set to 0, ZFS uses one-eighth of the available CPUs,
+with a minimum of 2 and a maximum of 16.
```

Comment:
Is there a time you would want both `dbuf_evict_threads >= 2` and `dbuf_evict_parallel=0`?  Just wondering if you can simplify this to `dbuf_evict_threads` only, and imply "parallel" if it's set to 2 threads or more.